tests/test_controller.py

```diff
@@ -1199,52 +1297,72 @@ def test_library_for_request_reloads_settings_if_necessary(self):
             # An assertion that would have failed before works now.
             assert new_name in self.manager.auth.library_authenticators
 
+    def test_load_lane(self):
+        # Verify that requests for specific lanes are mapped to
+        # the appropriate lane.
 
-class FullLaneSetupTest(CirculationControllerTest):
-    """Most lane-based tests don't need the full multi-tier setup of lanes
-    that we would see in a real site. We use a smaller setup to save time
-    when running the test.
```

Comment:
The test in this class was long out of date and not being run because its name didn't start with `Test`.